src/test/scala/org/broadinstitute/dsde/firecloud/MockServers.scala

```diff
@@ -136,6 +184,47 @@ object MockServers {
           )
           .withStatusCode(OK.intValue)
       )
+
+    // entity-level responses
+
+    MockServers.workspaceServer
+      .when(
+        request()
+          .withMethod("POST")
+          .withPath(s"/workspaces/${mockSampleValid.wsNamespace}/${mockSampleValid.wsName}/entities")
+          .withBody(mockSampleValid.toJson.compactPrint)
+          .withCookies(cookie)
+      ).respond(
+        response()
+          .withHeaders(header)
+          .withStatusCode(Created.intValue)
+      )
+
+    MockServers.workspaceServer
+      .when(
+        request()
+          .withMethod("POST")
+          .withPath(s"/workspaces/${mockPairValid.wsNamespace}/${mockPairValid.wsName}/entities")
+          .withBody(mockPairValid.toJson.compactPrint)
+          .withCookies(cookie)
+      ).respond(
+        response()
+          .withHeaders(header)
+          .withStatusCode(Created.intValue)
+      )
+
+    MockServers.workspaceServer
+      .when(
+        request()
+          .withMethod("POST")
+          .withPath(s"/workspaces/${mockSampleConflict.wsNamespace}/${mockSampleConflict.wsName}/entities")
+          .withBody(mockSampleConflict.toJson.compactPrint)
+          .withCookies(cookie)
+      ).respond(
+        response()
+          .withHeaders(header)
+          .withStatusCode(Conflict.intValue)
+      )
```

Comment:
The other approach that I think is worthwhile is moving mock server functionality to a common dsde library. Everyone could use the same mocked responses in that way. If we did have that, we could put common models there too. Entity, EntityCreateResult, Method, etc. should all be there so we might all be able to reduce duplicated efforts.